apps/extension/src/utils/index.ts

```diff
@@ -1,12 +1,58 @@
 import browser from "webextension-polyfill";
+import { v5 as uuid } from "uuid";
+
+import { DerivedAccount } from "@anoma/types";
+import { AccountStore } from "background/keyring";
 
 /**
- * Extension-specific utilities
+ * Query the current extension tab and close it
  */
-
 export const closeCurrentTab = async (): Promise<void> => {
   const tab = await browser.tabs.getCurrent();
   if (tab.id) {
     browser.tabs.remove(tab.id);
   }
 };
+
+/**
+ * Return all unencrypted values from account stores
+ */
+export const getAccountValuesFromStore = (
+  accounts: AccountStore[]
+): DerivedAccount[] => {
+  return accounts.map(
+    ({
```

Comment:
It would be great to add `pick` utility function or something :)

Reply:
I added a pick utility. I originally wanted to do an `omit` utility as well, but I think this way it's more clear what's being included, and we can add that later if it seems useful enough!